src/app/cube/content-pages/about-us/about-us.component.ts

```diff
@@ -0,0 +1,22 @@
+import { Component, ElementRef, OnInit, ViewChild } from '@angular/core';
+import { sections } from './copy';
+
+@Component({
+  selector: 'clark-about-us',
+  templateUrl: './about-us.component.html',
+  styleUrls: ['./about-us.component.scss']
+})
+export class AboutClarkComponent implements OnInit {
+
+  constructor() { }
+
+  ngOnInit(): void {
+  }
+
+  title = 'About Us';
+  copy = sections;
```

Comment:
Why are we setting a variable to an imported variable here?

Reply:
Because I over thought it. I'll get it fixed!

Reply:
This should be updated